File: src/main/java/br/com/zumbolovsky/fateapp/service/PersonagensService.java
package br.com.zumbolovsky.fateapp.service;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;
import br.com.zumbolovsky.fateapp.domain.Personagens;
import br.com.zumbolovsky.fateapp.domain.PersonagensRepository;

@Service
public class PersonagensService {

  @Autowired private PersonagensRepository personagensRepository;

  @Transactional
  public void insert(final Personagens personagens) {
    personagensRepository.save(personagens);
  }
}

File: src/main/java/br/com/zumbolovsky/fateapp/service/PersonagensService.kt
package br.com.zumbolovsky.fateapp.service

import br.com.zumbolovsky.fateapp.domain.Personagens
import br.com.zumbolovsky.fateapp.domain.PersonagensRepository
import org.springframework.stereotype.Service
import org.springframework.transaction.annotation.Transactional

@Service
class PersonagensService(private val personagensRepository: PersonagensRepository) {

    @Transactional
    fun insert(personagens: Personagens) {
        personagensRepository.save(personagens)
    }
}